test/CMakeLists.txt

```diff
@@ -2,5 +2,4 @@
 add_executable(testregex testregex.cpp)
 add_test(testregexcorrect testregex)
 
-
 # correctness testing of checks
```

Comment:
Actually we can modify the `add_new_check.py` script to add this line automaticaly when the check is created. So we would not have to worry about it.